pkg/trustedresources/verify.go

```diff
@@ -0,0 +1,195 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trustedresources
+
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/tektoncd/pipeline/pkg/apis/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// SignatureAnnotation is the key of signature in annotation map
+	SignatureAnnotation = "tekton.dev/signature"
+)
+
+// VerifyInterface get the checksum of json marshalled object and verify it.
+func VerifyInterface(
+	ctx context.Context,
+	obj interface{},
+	verifier signature.Verifier,
+	signature []byte,
+) error {
```

Comment:
I'm surprised the linter is okay with this formatting 🤔

Reply:
The lint will check gofmt and compare if the current code is of any difference with that. I make sure all code are formatted by gofmt so that's why it is not failing. 

So the issue is they should be in one line not multiple lines? If so I have reformatted them into one line.